libvmi/accessors.c

```diff
@@ -95,6 +95,34 @@ vmi_get_winver(
 #endif
 }
 
+uint16_t
+vmi_get_win_buildnumber(
```

Comment:
The only issue I see is that this right now only works if a JSON profile was provided, in all other cases the value returned would just be 0. I would add a safety check to see if LibVMI was initialized with a Json profile or not (check if `vmi->json.handler == NULL` and print an error if it wasn't.

Reply:
@tklengyel, done!